test/cpp/test_lazy.cpp

```diff
@@ -0,0 +1,122 @@
+#include <gtest/gtest.h>
+
+#include "tensorflow/compiler/xla/shape.h"
+#include "torch/csrc/lazy/core/shape.h"
+#include "torch_xla/csrc/helpers.h"
+#include "torch_xla_test.h"
+
+namespace torch_xla {
+namespace cpp_test {
+
+class LazyTest : public TorchXlaTest {};
+
+TEST_F(LazyTest, TestXlaShapeToLazyWithF64) {
+  int64_t dimensions[] = {1};
+  bool dynamic_dimensions[] = {false, false};
```

Comment:
Is this supposed to be the same size as `dimensions`? Just took a look over the XLA implementation. It doesn't seem they have any checks to ensure that. Just a comment. lol.

Reply:
Good point, yeah. It seems like they don't have any checks but it should be the same size.

Reply:
Updated the tests to have same sizes.